public/javascripts/SVValidate/src/modal/ModalMissionComplete.js

```diff
@@ -132,10 +133,14 @@ function ModalMissionComplete (uiModalMissionComplete, user, language = 'en') {
                 labelsValidated: mission.getProperty("labelsValidated")
             }
         );
+        if (svv.keyboard) {
```

Comment:
Your PR description said that all you did was comment out the extra `enableKeyboard()` call above. So did you leave this in by accident? Or was the description wrong and you also needed to disable the keyboard here?